carball/analysis/hit_detection/hit_analysis.py

```diff
@@ -127,6 +127,45 @@ def get_shot(data_frame: pd.DataFrame, saltie_hit: Hit, player_map: Dict[str, Pl
         if saltie_hit.goal and not is_shot:
             logger.warning('Goal is not shot: %s', saltie_hit)
 
+    @staticmethod
+    def get_clear(data_frame: pd.DataFrame, saltie_hit: Hit, next_saltie_hit: Hit,  player_map: Dict[str, Player]):
+        """
+        Finds clears based on distance travelled and positions.
+        :param game:
+        :param saltie_hit:
+        :param next_saltie_hit:
+        :param player_map:
+        """
+        CLEAR_BUFFER = 400
+        FIELD_LENGTH = 5120
+        defending = False
+        upfield_hit = False
+        player =  player_map[saltie_hit.player_id.id]
+
+        # get y-pos of player to determine if they are in their defending third
+        frame = saltie_hit.frame_number
+        y_pos = data_frame[player.name].iloc[frame]['pos_y']
+        if player.is_orange and y_pos > FIELD_LENGTH/3 + CLEAR_BUFFER:
+            defending = True
+        if not player.is_orange and y_pos < (-1)*FIELD_LENGTH/3 - CLEAR_BUFFER:
+            defending = True
+
+        # determine if hit passed the buffer to be considered a clear
+        if next_saltie_hit is not None:
+            # find next hit, determine if this hit went far enough
+            next_y = next_saltie_hit.ball_data.pos_y
+            if player.is_orange and next_y < FIELD_LENGTH/3 - CLEAR_BUFFER:
+                upfield_hit = True
+            if not player.is_orange and next_y > (-1)*FIELD_LENGTH/3 + CLEAR_BUFFER:
+                upfield_hit = True
+        else:
+            # a big hit to end the game should also count as a clear
+            distance = saltie_hit.distance
+            if distance > CLEAR_BUFFER:
+                upfield_hit = True
```

Comment:
This feels iffy. I'd rather still check that it goes to the neutral third at least. (so it will rule out a hit where the ball is hit **towards** his own goal).

Reply:
Without a succeeding hit, what would be the best way to determine if the ball reached the neutral third?